bitnami/supabase/values.yaml

```diff
@@ -950,6 +950,7 @@ realtime:
     DB_HOST: {{ include "supabase.database.host" . | quote }}
     DB_PORT: {{ include "supabase.database.port" . | quote }}
     DB_NAME: {{ include "supabase.database.name" . | quote }}
+    DB_USER: {{ include "supabase.database.user" . | quote }}
```

Comment:
What's the motivation of this changes? Seems to go further than just fix the public URL. Maybe unintentional changes?

Reply:
These changes aren't connected to the publicURL issue. I should have opend a pull request on it's own for this.

**The background of these change suggestions is:**
* Without knowing the `DB_USER` the realtime extension isn't able to write into the schema of it's extension (wheather it's called "realtime" or "_realtime" - in your deployment there exists at least a schema "realtime"). Instead realtime then writes it's information into the public schema, wich isn't what you want.
* After adding `DB_USER` you can add `DB_AFTER_CONNECT_QUERY: 'SET search_path TO <schema the extension shall use>'. Then it won't be using the public schema any more.
* Without specifying the `DNS_NODES` variable realtime can't be run it in a cluster.
* `ENABLE_TAILSCALE: "false"` actually shouldn't change anything because it sould be the default behavior.